cobald_hep_plugins/stopper.py

```diff
@@ -0,0 +1,65 @@
+from cobald.interfaces import Pool, PoolDecorator
+
+from cobald.utility import enforce
+
+import asyncio
+
+from cobald.daemon import service
+
+
+@service(flavour=asyncio)
+class Stopper(PoolDecorator):
+    """
+    Decorator that sets demand to 0 if the partition has no pending jobs
+
+    :param target: the pool
+    :param script: script that checks for pending jobs on the partition
```

Comment:
As far as I can tell, the code expects to be given the path to the script (not the name, code, ...).
```suggestion
    :param script: path to script that checks for pending jobs
```